src/components/Icon/index.js

```diff
@@ -18,22 +21,53 @@ const propTypes = {
 
     /** Is small icon */
     small: PropTypes.bool,
+
+    /** Is small icon */
+    inline: PropTypes.bool,
 };
 
 const defaultProps = {
     width: variables.iconSizeNormal,
     height: variables.iconSizeNormal,
     fill: themeColors.icon,
     small: false,
+    inline: false,
 };
 
+const inlineTop = Platform.OS === 'ios' ? 1 : 2;
+
 // We must use a class component to create an animatable component with the Animated API
 // eslint-disable-next-line react/prefer-stateless-function
 class Icon extends PureComponent {
     render() {
+        const inline = this.props.inline;
```

Comment:
Does not look like the need to extract in a variable. use `this.props.inline` directly.